libs/sdk-core/src/lib.rs

```diff
@@ -1,3 +1,143 @@
+//! locally.
+//!
+//! On initializing the SDK, the caller gets its [GreenlightCredentials]. These are used to interact
+//! with the Greenlight LN node running in the cloud. Together with the BIP39 mnemonic, these can be used to
+//! restore access to the same cloud node, either in the same app (backup / restore) or in another app
+//! using the SDK.
+//!
+//! In addition, the SDK includes utilities for LSP management, on-chain payments and fiat on- and
+//! off-ramps.
+//!
+//! ## Getting Started
+//!
+//! The first step is to initialize the SDK and start the node:
+//!
+//! ```ignore
+//! let mnemonic = Mnemonic::new(Words12, English);
+//! let seed = Seed::new(&mnemonic, "");
+//!
+//! let creds = BreezServices::register_node(Network::Bitcoin, seed.as_bytes().to_vec()).await?;
+//! let sdk = BreezServices::init_services(
+//!         None,
+//!         seed.to_vec(),
+//!         creds.clone(),
+//!         Box::new(AppEventListener {}),
+//!     )
+//!     .await?;
+//!
+//! BreezServices::start(rt(), &sdk).await?;
+//! ```
+//! We can now receive payments
+//!
+//! ```ignore
+//! let invoice = sdk.receive_payment(3000, "Invoice for 3000 sats".into()).await?;
+//! ```
+//!
+//! or make payments
+//! ```ignore
+//! let bolt11 = "..."; // LN invoice
+//! sdk.send_payment(bolt11.into(), Some(3000)).await?;
+//! ```
+//!
+//! At any point we can fetch our balance from the Greenlight node
+//! ```ignore
+//! if let Some(node_state) = sdk.node_info()? {
+//!     let balance_ln = node_state.channels_balance_msat;
+//!     let balance_onchain = node_state.onchain_balance_msat;
+//! }
+//! ```
+//!
+//! or fetch other useful infos, like the current mempool [RecommendedFees]
+//! ```ignore
+//! let fees: RecommendedFees = sdk.recommended_fees().await?;
+//! ```
+//!
+//! These different types of operations are described below in more detail.
+//!
+//! ### A. Initialize the SDK
+//!
+//! There are two steps necessary to initialize the SDK:
+//!
+//! 1. [BreezServices::init_services] to setup the Breez SDK services
+//! 2. [BreezServices::start] to start the Greenlight node and all needed Breez SDK services
+//!
+//! The first step takes the [GreenlightCredentials] as an argument. There are three ways to get them:
+//!
+//! * by loading the credentials from local storage, or with
+//! * [BreezServices::register_node] to register a new Greenlight node, or with
+//! * [BreezServices::recover_node] to recover an existing Greenlight node
+//!
+//! After initializing the Breez SDK services and starting them, the SDK is ready to be used.
+//!
+//! ### B. LN Operations
+//!
+//! Supported BOLT11 operations are
+//!
+//! * [BreezServices::receive_payment] to create an invoice
+//! * [BreezServices::send_payment] to pay an invoice
+//!
+//! Keysend is possible with [BreezServices::send_spontaneous_payment].
+//!
+//! See LNURL section below for LNURL-pay and -withdraw.
+//!
+//! ### C. On-chain Operations
```

Comment:
I think it is better to have a category for On-chain Receive (swap in) with the following methods:
[BreezServices::receive_onchain]
[BreezServices::list_refundables] for a list of swaps
[BreezServices::refund]

Reply:
Done. I then moved `[BreezServices::sweep]` to the Wallet Utilities section.